image/image.css

```diff
@@ -0,0 +1,39 @@
+:root{
+    --shadow-color: #b8b8b3d2;
+}
+
+.sidebar-img{
+    width: 50%;
+}
+.container{
+    padding: 1rem;
+    gap: 2rem;
+}
+.component-container{
+    padding: 2rem 4rem 2rem 2rem;
+}
+.component-details{
+    padding: 1rem;
+}
+
+.img-container{
+    width: 90%;
+}
+
+.img-example {
+    justify-content: center;
+    align-items: center;
+    padding: 2rem;
+}
+.img-responsive{
+    width: 100%;
+    height: auto;
+    box-shadow: 0 5px  5px var(--shadow-color), 0 6px 20px 0 var(--shadow-color);
```

Comment:
add an object-fit property to the img-responsive class because when I am increasing the height of the div above the image tag, the image is not adjusting to it.

![Screenshot 2022-02-12 at 9 50 47 AM](https://user-images.githubusercontent.com/79623546/153696355-08d3bc9d-5c0d-44f0-83dc-041f412d469e.png)

Reply:
+1

Reply:
Will do, thanks